src/Combinatorics/Matroids/quantum_automorphism_groups.jl

```diff
@@ -1,8 +1,93 @@
+
+
+function _index_number(i::Int)
+  dgs = reverse(digits(i))
+  return join(["₀₁₂₃₄₅₆₇₈₉"[3*d+1] for d in dgs], "") 
```

Comment:
```suggestion
  ind = ['₀', '₁', '₂', '₃', '₄', '₅', '₆', '₇', '₈', '₉']
  return join(ind[d] for d in dgs) 
```
This seems slightly faster and the reader doesn't need to know that these are 3-byte characters.

Or maybe even this:
```suggestion
  return join('₀'+d for d in dgs)
```